azure-pipelines.yml

```diff
@@ -262,14 +267,11 @@ jobs:
       # copy the rom to a specific location
       ROM_TARGET="${BIN_DIR}/sw/device/silicon_creator/rom"
       mkdir -p "${ROM_TARGET}"
-      AUTHENTICITY=(fake real)
-      for authenticity in "${AUTHENTICITY[@]}"; do
-        ROM=$(ci/scripts/target-location.sh "//sw/device/silicon_creator/rom:rom_with_${authenticity}_keys_fpga_cw310_scr_vmem")
-        cp $ROM "${ROM_TARGET}/$(basename "${ROM}")"
-        ROM_DIR=$(dirname $ROM)
-        cp "${ROM_DIR}/rom_with_${authenticity}_keys_fpga_cw310.elf" "${ROM_TARGET}/"
-        cp "${ROM_DIR}/rom_with_${authenticity}_keys_fpga_cw310.bin" "${ROM_TARGET}/"
-      done
+      # The deps 2 levels down from the opentitan_rom_binary include all the
+      # important files.
+      ROM_REAL_FILES=($(./bazelisk.sh outquery-all --noinclude_aspects --noimplicit_deps "deps(//sw/device/silicon_creator/rom:rom_with_real_keys_from_src_fpga_cw310, 2)" | sort | uniq))
```

Comment:
This took some experimentation... To make the correct query to get the ELF output file, we have to start from a node that is built for our usual target platform (the host) and follow the edges in the graph across the configuration transition (to riscv32).

Reply:
> To make the correct query to get the ELF output file, we have to start from a node that is built for our usual target platform (the host) and follow the edges in the graph across the configuration transition (to riscv32).

Maybe we can add this explanation as a comment in the code?

Reply:
Done.